src/txn/txn_rollback_to_stable.c

```diff
@@ -1623,6 +1623,18 @@ __rollback_progress_msg(WT_SESSION_IMPL *session, struct timespec rollback_start
     }
 }
 
+/*
+ * __check_btree_dirty --
+ *     Check that the rollback to stable btree is dirty or not.
+ */
+static int
+__check_btree_dirty(WT_SESSION_IMPL *session, const char *uri, bool *perform_rts)
```

Comment:
Would it make more sense if the output variable was `*modified` or 
`*dirty` instead of `*perform_rts`? I think it would better match the function's name.

Reply:
I agree to change it along with the function name. `modified` variable name with the `__check_btree_modified` function name.